shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -491,3 +447,272 @@ - (void)insertViewToPoolIfNeeded:(FlutterClippingMaskView*)maskView {
 }
 
 @end
+
+@implementation UIView (FirstResponder)
+- (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
```

Comment:
Should it just be "FLTHasFirstResponder..."?

Reply:
Just `hasFirstResponderInViewHierarchySubtree` since it's a method. There shouldn't be any need for an `flt` prefix at all.

Reply:
I believe the prefix is correct since it's a category on a UIKit class.

> Methods in a category should be prefixed with a lowercase version of the prefix used for the category name followed by an underscore (e.g., gtm_myCategoryMethodOnAString:) in order to prevent collisions in Objective-C’s global namespace.

https://google.github.io/styleguide/objcguide.html#category-naming